gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/conn/GenericStatementContext.java

```diff
@@ -784,11 +784,13 @@ public StringBuilder appendErrorInfo() {
 
 		StringBuilder sb = ((ContextImpl) lcc).appendErrorInfo();
 		if (sb != null) {
-
-			sb.append("Failed Statement is: ");
-
-			sb.append(getStatementText());
-
+			String stmtText = getStatementText();
```

Comment:
This only skips `Failed Statement is: ` part of the log messages. We need to take care of entire log message.
Returning a null value of `sb` might cover the entire message.

But for the second log message (`Lexical error` or `Syntax error`), you may want to take a look at a for loop label `forever` in `ContextManager.java` . There you can skip adding `error.toString()` to the `errorStringBuilder` if the SQL is one of these three.

Reply:
Resolved. Moved the If statement into ContextManager.java from GenericStatementContext.java